test/e2e/locators/recordset.ts

```diff
@@ -22,6 +22,19 @@ export type TimestampRangeInputLocators = {
   submit: Locator;
 }
 
+export type HistogramLocators = {
+  zoom: Locator;
+  zoomDisabled: Locator;
+  unzoom: Locator;
+  unzoomDisabled: Locator;
+  reset: Locator;
+}
+
+export type TimestampMinMax = {
```

Comment:
This should be `TimestampDateTime`. That's the name that you've used in the spec.